config/master.d/50-returner.conf

```diff
@@ -5,71 +5,4 @@ mysql:
   user: 'salt'
-  db: 'velum_production'
-
-ext_pillar:
-  - mysql:
-    # This queries allows us to override our pillar values up to a depth of 3.
-    #
-    # Considering we have on our static pillar:
-    #
-    #   certificate_information:
-    #     subject_properties:
-    #       C: DE
-    #       O: SUSE
-    #       ...
-    #
-    # We are able to prioritize pillar information overriding the default values.
-    #
-    # In this example, if we had a mysql row:
-    #   INSERT INTO pillars(pillar, value) VALUES ("certificate_information:subject_properties:O",
-    #                                              "My Company");
-    #
-    # We would see on the pillar:
-    #
-    #   certificate_information:
-    #     subject_properties:
-    #       C: DE
-    #       O: My Company
-    #       ...
-    #
-    # Please, note that this will only work until a depth of 3 properties. If we want to provide
-    # more depth, we would need to add more columns, as salt treats each column as a nested property
-    # inside the previous one. For further information, please visit:
-    #   https://docs.saltstack.com/en/latest/ref/pillar/all/salt.pillar.sql_base.html
-    #
-    # Lists can be represented as collisions:
-    #   INSERT INTO pillars(pillar, value) VALUES ("my_service:extra_ips", "127.0.0.1");
-    #   INSERT INTO pillars(pillar, value) VALUES ("my_service:extra_ips", "127.0.0.2");
-    #
-    # Will be represented as:
-    #
-    #   my_service:
-    #     extra_ips:
-    #       - 127.0.0.1
-    #       - 127.0.0.2
-    #
-    # Query for first level pillar values (no dots in `pillar` key)
-    - query: 'SELECT pillar,
-                     value
-                     FROM pillars WHERE
-                       (LENGTH(pillar) - LENGTH(REPLACE(pillar, ":", "")) = 0) AND
-                       (minion_id IS NULL OR minion_id = %s)'
-      as_list: True
-    # Query for second level pillar values (one dot in `pillar` key)
-    - query: 'SELECT SUBSTRING_INDEX(SUBSTRING_INDEX(pillar, ":", 1), ":", -1) AS key1,
-                     SUBSTRING_INDEX(SUBSTRING_INDEX(pillar, ":", 2), ":", -1) AS key2,
-                     value
-                     FROM pillars WHERE
-                       (LENGTH(pillar) - LENGTH(REPLACE(pillar, ":", "")) = 1) AND
-                       (minion_id IS NULL OR minion_id = %s)'
-      as_list: True
-    # Query for third level pillar values (two dots in `pillar` key)
-    - query: 'SELECT SUBSTRING_INDEX(SUBSTRING_INDEX(pillar, ":", 1), ":", -1) AS key1,
-                     SUBSTRING_INDEX(SUBSTRING_INDEX(pillar, ":", 2), ":", -1) AS key2,
-                     SUBSTRING_INDEX(SUBSTRING_INDEX(pillar, ":", 3), ":", -1) AS key3,
-                     value
-                     FROM pillars WHERE
-                       (LENGTH(pillar) - LENGTH(REPLACE(pillar, ":", "")) = 2) AND
-                       (minion_id IS NULL OR minion_id = %s)'
-      as_list: True
+  db: 'velum_production'
```

Comment:
What editor do you use? I'm always confused with this "No newline at the enf of file"